File: pinyougou_manager_web/src/main/webapp/js/service/brandService.js
app.service("brandService",function ($http) {
    this.search=function (pageNum,pageSize,searchEntity) {//分页条件查询
        return   $http.post('../brand/search.do?pageNum='+pageNum+'&&pageSize='+pageSize,searchEntity)
    }
    this.deleteBrand=function (selectIds) {
        return $http.post('../brand/delete.do?ids='+selectIds)
    }
    this.findOne=function (id) {
        return $http.post('../brand/findbyid.do?id='+id)
    }
    this.saveBrand=function (entity) {


        return  $http.post('../brand/savebrand.do',entity)
    }
    this.updateBrand=function (entity) {

        return $http.post('../brand/update.do',entity)
    }
    this.findAll=function () {

        return  $http.post('../brand/findAll.do')
    }
    this.selectOptionList=function(){
        return $http.get('../brand/selectOptionList.do');
    }

})

File: pinyougou_sellergoods_interface/src/main/java/com/scott/serllergoods/service/BrandService.java
package com.scott.serllergoods.service;

import com.scott.pojo.TbBrand;
import entity.PageResult;

import java.util.List;
import java.util.Map;

/**
 * @author scott
 * @version 1.0
 * @description com.scott.serllergoods.service
 * @date 2018/4/21
 */
public interface BrandService {
    public List<TbBrand> findAll();
    public PageResult findPage(int pageNum,int pageSize);
    public void saveBrand(TbBrand tbBrand);
    public TbBrand findById(Long id);
    public void update(TbBrand tbBrand);
    public  void deleteBrand(Long[] ids);
    public PageResult search(TbBrand tbBrand,int pageNum,int pageSize);

    List<Map> selectOptionList();
}
